examples/python/helloworld/greeter_client_with_options.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Python implementation of the GRPC helloworld.Greeter client."""
+
+from __future__ import print_function
+
+import grpc
+
+import helloworld_pb2
+import helloworld_pb2_grpc
+
+
+def run():
+    # NOTE(gRPC Python Team): .close() is possible on a channel and should be
+    # used in circumstances in which the with statement does not fit the needs
+    # of the code.
+    #
+    # For more channel options, please see https://grpc.io/grpc/core/group__grpc__arg__keys.html
+    with grpc.insecure_channel(
+            target='localhost:50051',
+            options=[('grpc.lb_policy_name', 'pick_first'),
+                     ('grpc.enable_retries', 0),
+                     ('grpc.keepalive_timeout_ms', 10)]) as channel:
```

Comment:
10 is a very low value which can lead to unnecessary timeout. Default is 20 sec and we don't recommend less than 10 sec. So please change this to 10000. We don't want users to follow this example and get into trouble.